aspnetcore/security/authentication/social/facebook-logins.md

```diff
@@ -142,7 +117,7 @@ You are now logged in using your Facebook credentials:
 
 ## Next steps
 
-* Add the [Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook) NuGet package to your project for advanced Facebook authentication scenarios. This package isn't required to integrate Facebook external login functionality with your app. 
+* Add the [Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook) NuGet package to your project for advanced Facebook authentication scenarios. This package isn't required to integrate Facebook external login functionality with your app.
```

Comment:
Delete this line. You have them add it at the start of the tutorial. It used to be included but no longer is.